src/frameworks.sources

```diff
@@ -225,6 +225,7 @@ AUDIOUNIT_SOURCES = \
 AUTHENTICATIONSERVICES_SOURCES = \
 	AuthenticationServices/ASAuthorization.cs \
 	AuthenticationServices/ASAuthorizationRequest.cs \
+	PublicPrivateKeyAuthentication.cs \
```

Comment:
I think fixing this typo should help you get closer!
```suggestion
	AuthenticationServices/PublicPrivateKeyAuthentication.cs \
```

Reply:
Completely missed this haha thank you!